pkgs/tools/text/termbook/default.nix

```diff
@@ -11,12 +19,37 @@ rustPlatform.buildRustPackage rec {
     sha256 = "Bo3DI0cMXIfP7ZVr8MAW/Tmv+4mEJBIQyLvRfVBDG8c=";
   };
 
-  cargoSha256 = "sha256-9fFvJJlDzBmbI7hes/wfjAk1Cl2H55T5n8HLnUmDw/c=";
+  cargoLock = {
+    lockFile = ./Cargo.lock;
+  };
+
+  nativeBuildInputs = [
+    installShellFiles
+    pkg-config
+  ];
 
-  buildInputs = lib.optionals stdenv.isDarwin [
+  buildInputs = [
+    oniguruma
+  ] ++ lib.optionals stdenv.isDarwin [
     darwin.apple_sdk.frameworks.Security
   ];
 
+  env = {
+    RUSTONIG_SYSTEM_LIBONIG = true;
```

Comment:
We can't set this var and the openssl no vendor for the rust wrapper?

Reply:
maybe, but I would leave this out of this PR (and probably 23.05)